src/Tools/ProcessWatchdog/ConsoleUtils.cs

```diff
@@ -13,14 +13,20 @@ internal static void LogMessage(string format, params object[] args)
                 string.Format(CultureInfo.CurrentCulture, format, args));
         }
 
-        internal static void LogError(string messageFormat, params object[] args)
+        internal static void LogError(ErrorCode errorCode, string messageFormat, params object[] args)
         {
             string fullMessage = string.Format(
-                CultureInfo.CurrentCulture,
+                CultureInfo.InvariantCulture,
```

Comment:
The overall error message format "error PW000n: Some message" needs to be invariant, because a program will scrape it. The human-readable message can be localized.